packages/core/content-type-builder/admin/src/translations/fi.json

```
@@ -0,0 +1,208 @@
+  "error.contentType.singularName-used": "Tämä arvo ei voi olla sama kuin monikko",
+  "error.contentType.singularName-equals-pluralName": "Tämä arvo ei voi olla sama kuin toisen sisältötyypin Plural API ID.",
+  "error.contentType.pluralName-equals-singularName": "Tämä arvo ei voi olla sama kuin toisen sisältötyypin Singular API ID.",
+  "error.contentType.pluralName-equals-collectionName": "Tämä arvo on jo toisen sisältötyypin käytössä.",
+  "error.contentTypeName.reserved-name": "Tätä nimeä ei voi käyttää projektissasi, koska se saattaa rikkoa muita toimintoja",
+  "error.validation.enum-duplicate": "Kaksoisarvot eivät ole sallittuja (vain aakkosnumeeriset merkit otetaan huomioon).",
+  "error.validation.enum-empty-string": "Tyhjät merkkijonot eivät ole sallittuja",
+  "error.validation.enum-regex": "Ainakin yksi arvo on virheellinen. Arvoissa tulee olla vähintään yksi aakkosmerkki ennen luvun ensimmäistä esiintymää.",
+  "error.validation.minSupMax": "Minimi ei voi olla korkeampi kuin maksimi",
+  "error.validation.positive": "On oltava positiivinen luku",
+  "error.validation.regex": "Regex-malli on virheellinen",
+  "error.validation.relation.targetAttribute-taken": "Tämä nimi on olemassa kohteessa",
+  "form.attribute.component.option.add": "Lisää komponentti",
+  "form.attribute.component.option.create": "Luo uusi komponentti",
+  "form.attribute.component.option.create.description": "Osa on jaettu tyypeille ja komponenteille, se on saatavilla ja käytettävissä kaikkialla.",
+  "form.attribute.component.option.repeatable": "Toistettava komponentti",
+  "form.attribute.component.option.repeatable.description": "Paras useille ainesosien, sisällönkuvauskenttien jne. esiintymille (matriisille).",
+  "form.attribute.component.option.reuse-existing": "Käytä olemassa olevaa komponenttia",
+  "form.attribute.component.option.reuse-existing.description": "Käytä uudelleen jo luotua komponenttia pitääksesi tietosi yhtenäisinä eri sisältötyypeissä.",
+  "form.attribute.component.option.single": "Yksi komponentti",
+  "form.attribute.component.option.single.description": "Paras kenttien ryhmittelyyn, kuten täydellinen osoite, päätiedot jne...",
+  "form.attribute.item.customColumnName": "Muokatut sarakkeiden nimet",
+  "form.attribute.item.customColumnName.description": "Tämä on hyödyllistä nimetä tietokannan sarakkeiden nimet uudelleen kattavampaan muotoon API-vastauksia varten",
+  "form.attribute.item.date.type.date": "päivämäärä (esim: 01/01/{currentYear})",
+  "form.attribute.item.date.type.datetime": "datetime (esim: 01/01/{currentYear} 00:00 AM)",
+  "form.attribute.item.date.type.time": "aika (esim. 00:00 AM)",
+  "form.attribute.item.defineRelation.fieldName": "Kentän nimi",
+  "form.attribute.item.enumeration.graphql": "GraphQL:n nimen ohitus",
+  "form.attribute.item.enumeration.graphql.description": "Voit ohittaa GraphQL:lle luodun oletusnimen",
+  "form.attribute.item.enumeration.placeholder": "Esi.:\naamu\niltapäivä\nilta",
```

```suggestion
  "form.attribute.item.enumeration.placeholder": "Esi.:\naamu\niltapäivä\nilta",
```
Should this be `Esim.`? What is the original string?

---

yes i think so. The english for some reason uses 'ex.', rather than eg, weirdly